src/main/java/org/springframework/hateoas/AffordanceModel.java

```diff
@@ -149,8 +149,8 @@ public PayloadMetadata getOutput() {
 	 */
 	public <T> List<T> createProperties(BiFunction<InputPayloadMetadata, PropertyMetadata, T> creator) {
 
-		return input.stream()
-				.map(it -> creator.apply(input, it))
+		return input.stream() //
+				.map(it -> creator.apply(input, it)) //
```

Comment:
I'm curious, what's the use of "//"?

Reply:
In a fluent API, in which you chain method calls, we use those to prevent the IDE from reformatting the code so that we end up with each invocation on a separate line.

Reply:
Being managed by the Spring Data team, we format code on the backend via https://github.com/spring-projects/spring-data-build/tree/main/etc/ide.

We don't lean on maven plugins to reformat code nor ask that contributors "polish code" to fashion a pull request. Thus, this is how we maintain things.

Reply:
Thanks for your detailed explanation.